wandb/sdk/wandb_helper.py

```diff
@@ -4,20 +4,21 @@
 
 import six
 from wandb.errors.error import UsageError
+from wandb.lib import config_util
 
 
 def parse_config(params, exclude=None, include=None):
     if exclude and include:
         raise UsageError("Expected at most only one of exclude or include")
-
+    if isinstance(params, str):
+        params = config_util.dict_from_config_file(params)
```

Comment:
isnt params going to get blown away in the next line

Reply:
and looks like we just get a log (debug level ) that the file didnt exist, but i htink the wandb.init() call should fail

Reply:
ignore the "params blown away" comment, file not found needs still needs to be addressed.